### ISSUE TITLE
GNOME 3.6 support

### ISSUE THREAD

**Issue by ai** [crantisz/Layout-swicher-seach-ru-en#1]
Awesome extenstion, I miss it in GNOME 3.6 :).

**Comment by crantisz** [comment]
I can't understand  how to debug this in 3.6 - cant find some functional. Maybe you can find anybody to help this it?

**Comment by ai** [comment]
I can try on weekend. What exactly you need?

**Comment by ai** [comment]
By the way, [Advanced Calculator](https://extensions.gnome.org/extension/121/gcalcsearch/) can be useful for you, because it modify search query too.

**Comment by crantisz** [comment]
just debug and restore functional. I can't find error tab in lg, so can't see any errors and logs

**Comment by ai** [comment]
Yeah, debug is real big problem in GNOME :).

Error log is in `~/.cache/gdm/session.log`.

**Comment by crantisz** [comment]
I cant find anything in this file - and i need logs/

**Comment by ai** [comment]
OK, if you can enable extension, so there isn’t any errors during loading.

**Comment by ai** [comment]
I try to debug it now.

**Comment by ai** [comment]
Do you see fix comment in https://extensions.gnome.org/extension/34/layout-swicher-seach-ru-en/ from nilus?

**Comment by crantisz** [comment]
Yes, but it doesn't work. And i removed all errors, but it works wrong, and i can't fix it without logs

**Comment by ai** [comment]
You can write `print("text")` and result will be in `~/.cache/gdm/session.log`.

**Comment by ai** [comment]
Also you can call `Main.notify("text")`, which will show nice notify with you debug text.

**Comment by ai** [comment]
Don’t forget to restart GNOME Shell after every changes in extesion script: Alt+F2, type `r` and press Enter.

**Comment by crantisz** [comment]
Yea, I know. in 3.4 and 3.2 everything was good/

**Comment by crantisz** [comment]
print("text") - isn;t work
Main.notify - can't see anything

**Comment by crantisz** [comment]
hm, ok all works, it just stops on some error...
